docs/multiindex_isotope_decay_data.ipynb

```diff
@@ -0,0 +1,446 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    import astropy.units as u
</pre><p>Do we need all these imports?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Removed.